OADP-4058: Adding structure for OADP and 3scale user story (#85994)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/oadp-3scale/backing-up-and-restoring-3scale-using-oadp.adoc

:_mod-docs-content-type: ASSEMBLY
[id="backing-up-and-restoring-3scale-using-oadp"]
= Backing up and restoring 3scale using OADP
include::_attributes/common-attributes.adoc[]
:context: backing-up-and-restoring-3scale-using-oadp

toc::[]

With Red Hat 3scale API Management (APIM), you can manage your APIs for internal or external users. Share, secure, distribute, control, and monetize your APIs on an infrastructure platform built with performance, customer control, and future growth in mind.
3scale components can be deployed on-premise, in the cloud, as a managed service, or in any combination based on your requirement.

> This document is using the non-service affecting approach to backup and restore 3scale on cluster storage using the {oadp-first} Operator. Here, either the source or destination cluster are the same or if not, both the clusters are using the same custom domain.

Reviewer: ```suggestion
This document is using the non-service affecting approach to backup and restore 3scale on cluster storage by using the {oadp-first} Operator. Here, either the source or destination clusters are the same or both clusters are using the same custom domain.
```
Is it possible to get rid of "This document"? We're advised to avoid self-ref text as much as possible? Maybe there's a good way to rephrase this sentence?
Also, the second sentence is a bit hard to understand (for me, at least, hehe :-D). Any idea on how to rephrase it? (Not a requeitherirement, just a  suggestion)

Author: Added it as a note - In this case, the non-service affecting approach is used to backup and restore 3scale on cluster storage by using the {oadp-first} Operator.

In addition to that, either use the same source or destination cluster. Otherwise, ensure that both the cluster are using the same custom domain
